workflows/Get_InsightVM_Host_Info_From_Slack/workflow.spec.yaml

```
@@ -0,0 +1,25 @@
+extension: workflow
+products: ["insightconnect"]
+name: Get_InsightVM_Host_Info_From_Slack
+title: "Get InsightVM Host Info from Slack"
+description: Fetch asset details in seconds from InsightVM using a Slack command. Host info includes Hostname, IP, MAC, OS, services, and vulnerability stats.
```

```suggestion
description: This workflow provides fast, convenient access to information about a given host from InsightVM. When triggered, the workflow fetches asset details from InsightVM and posts them in a Slack thread. Host info includes hostname, IP address, MAC address, operating system, running services, and vulnerability stats.
```
I think it should match the help.md description.